Documentation/anti-patterns.md

```diff
@@ -38,6 +38,7 @@ connection until an RPC is executed. Instead of using the WithBlock option, whic
 may not be recommended in some cases, you can call the
 [`ClientConn.Connect`](https://pkg.go.dev/google.golang.org/grpc#ClientConn.Connect)
 method to explicitly initiate a connection.
+[`WithBlock`](https://pkg.go.dev/google.golang.org/grpc#WithBlock) in an instance where WithBlock(true) is used, `Connect` and `WaitForStateChange` is invoked until either the context created via `context.WithTimeout` expires or the `ClientConn` is ready.
```

Comment:
Do we need to add this here since `WithBlock` is deprecated, and is not available in NewClient?